### ISSUE TITLE
Drink: Cassis missing recipe

### ISSUE THREAD

**Issue by Dockter** [AlmuraDev/Almura#222]
Item:  Almura_DrinksSpritit_CASSIS - 4309:0

**Comment by mcsnetworks** [comment]
Brandy is also missing....

**Comment by Wolfeyeamd0** [comment]
I made a list of the ones I couldn't find a recipe for, (making a cocktail bar is hard without some of these):

Cassis Liquer: ALMURA_DRICKSSPIRIT_CASSIS - 4309:0
Fine Brandy: ALMURA_DRINKSSPIRIT_BRANDY - 4308:0
Triple Sec:  ALMURA_DRINKSSPIRIT_TRIPLESEC - 4317:0
Vermouth:  ALMURA_DRINKSSPIRIT_VERMOUTH - 4318:0
Snake Juice:  ALMURA_DRINKSOTHER_SNAKEJUICE - 4304:0
Flask of Rotgut: ALMURA_DRINKSOTHER_ROTGUT - 4303:0 
Flask of Firewater:  ALMURA_DRINKSOTHER_FIREWATER - 4298:0
Jug of Moonshine:  ALMURA_DRINKSOTHER_MOONSHINE - 4302:0
Refined Port:  ALMURA_DRINKSSPIRIT_PORT - 4313:0
Aged Sherry: ALMURA_DRINKSSPIRIT_SHERRY - 4315:0
Greenwood Priory Fine Red Wine:  ALMURA_DRINKSWINE_RED_PRIORY - 4324:0
Greenwood Priory Fine Rose Wine:  ALMURA_DRINKSWINE_ROSE_PRIORY - 4330:0
Greenwood Priory Fine White Wine:  ALMURA_DRINKSWINE_WHITE_PRIORY - 4336:0
Greenwood Priory Special Vintage 84:  ALMURA_DRINKSWINE_RED_PRIORY84 - 4326:0
*Rreenwood Priory Special Vintage 79:  ALMURA_DRINKSWINE_RED_PRIORY79 - 4325:0

-------------------------------------------------
* Rreenwood should be spelled Greenwood.  This has been wrong for a long time, just keep forgetting about it until I look into the drinks.
---------------------------------------------------

I have set up shops in the Grocery Store for these for the time being.  We can always break them when we get recipes.
